go.mod

```diff
@@ -105,9 +105,9 @@ require (
 	golang.org/x/crypto v0.0.0-20190829043050-9756ffdc2472
 	golang.org/x/net v0.0.0-20190827160401-ba9fcec4b297
 	golang.org/x/oauth2 v0.0.0-20190604053449-0f29369cfe45
-	golang.org/x/sys v0.0.0-20190902133755-9109b7679e13
+	golang.org/x/sys v0.0.0-20190904005037-43c01164e931
```

Comment:
Why does it have to be updated?

Reply:
@typeless Only because of CI failing again. It's not this PR, is all of them.
For example: #8040 --> https://drone.gitea.io/go-gitea/gitea/12635

Reply:
@typeless please see #8076 - that PR contains **only** the vendor update.